lecture/models.py

```
@@ -1,26 +1,55 @@
+        Course,
+        on_delete=models.CASCADE,
+        related_name='lectures'
+    )
+    module = models.ForeignKey(
+        Module,
+        on_delete=models.CASCADE,
+        related_name='lectures'
+    )
+
+    title = models.CharField(max_length=255)
+    description = models.TextField(null=True, blank=True)
```

Ruff (0.8.2)

Avoid using `null=True` on string-based fields such as `TextField`

(DJ001)